tests/test_waveforms.py

```diff
@@ -301,6 +301,15 @@ def test_interpolated():
         ),
     ):
         InterpolatedWaveform(duration, [0, 0.1, 0.2, 0.3], other_values)
+    times = seq.declare_variable("times", size=6)  # a Variable
```

Comment:
Try this and see if it still works (my guess is that it won't)
```suggestion
    times = seq.declare_variable("times", size=6) + 1
```

Reply:
I know it won't ;)